README.md

```diff
@@ -37,7 +42,7 @@ We have released the GDK for Unreal this early in development because we want yo
 The GDK and its starter projects are available on GitHub.
 * [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
 * [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
-* [Third Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
+* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
 * [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
 * [The SpatialOS Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) </br>
```

Comment:
For consistency with my comments in the other README docs, we could move this up so it's just below the GDK bullet point. (Or, if you didn't make that change in the other docs, leave it here as well.)